package.json

```diff
@@ -81,7 +84,7 @@
   "bundlesize": [
     {
       "path": "./build/ddocs/medic/_attachments/js/inbox.js",
-      "maxSize": "800 kB"
+      "maxSize": "820 kB"
```

Comment:
The build is failing on this new size so it needs to be bumped further. Again, why has this increased?

Reply:
This is consistent with master - ignoring.